doc/index.rst

```diff
@@ -254,6 +258,54 @@ The jobs that run Prerelease Test may usually take longer than the tests defined
 
 Then open a pull request using this branch against the branch that the change is subject to be merged. You do not want to actually merge this branch no matter what the Travis result is. This branch is solely for Prerelease Test purpose.
 
+ABI checks
+----------
+
+Generally speaking, the `ABI <https://en.wikipedia.org/wiki/Application_binary_interface>`_ of a library can break for various reasons. A detailed explanation and a list of DOs and DON'Ts can be found in the `KDE Community Wiki <https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B>`_.
+
+The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to the **stable version** of your code.
+
+ABI check example configs
++++++++++++++++++++++++++
+
+Simplest example: Check against a specific stable branch (e.g. `kinetic` branch) for push and pull request tests:
+::
+  - ROS_DISTRO=kinetic
+    ABICHECK_URL='github:ros-industrial/ros_canopen#kinetic'
+
+If pull requests should be checked against the merge parent instead of the stable version (travis only):
```

Comment:
The only benefit is that PR might pass even if the target branch breaks the ABI to the stable version.